docs/services/firezone.md

```diff
@@ -0,0 +1,23 @@
+# Firezone
+
+[Firezone](https://www.firezone.dev/) is a self-hosted VPN server with Web UI that this playbook can install using the ansible role [moan0s/role-firezone](https://github.com/moan0s/role-firezone).
+
+To enable Firezone add the following to your `vars.yml`:
+
+```yaml
+##############
+## FIREZONE ##
+##############
+
+firezone_enabled: true
+firezone_hostname: example.org
+
+firezone_default_admin_email: "user@invalid.org"
+firezone_default_admin_password: "<securepassword>"
+
+# Generate this with `openssl rand -base64 32`
+firezone_database_encryption_key: "<secret>"
+```
+
+Use `ansible-playbook -i inventory/hosts setup.yml --tags=firezone-create-or-reset-admin` to create the configured
+admin account or reset the password to the password set in `vars.yml`.
```

Comment:
It might be worth adding a **Networking** subsection in **Configuration** which tells people what ports will be opened and potentially need to be forwarded if they're using another firewall, similar to bcebaafb29a9b8cf314d1.


A **Usage** section (like the one we have in most other service docs) may be helpful too.

Reply:
I added both sections (but kept Usage short, as the upstream documentation is quite good I think)